### PR TITLE
[CoW] Migrate cow.solvers table

### PR DESCRIPTION
**Pull request by bh2smith**
Migration of Cow Protocol Solvers Table(s) -- both ethereum and gnosis chain.

Updated Unit tests here: https://dune.com/queries/2718501 Will add link in the test file.

cc @gentrexha

### CODE REVIEW

**Comment by bh2smith** [comment]
Just wanted to note that [this PR](https://github.com/duneanalytics/spellbook/pull/3726) was approved and merged into this branch but somehow got overwritten during one of the rebase/squash commits. Sorry about that, just want to ensure its references here.

**Review by jkylling — APPROVED** [review]
We need to verify that the data matches between the migrated and the legacy spells.

File: models/cow_protocol/gnosis/cow_protocol_gnosis_solvers.sql [hunk 1]
```diff
@@ -1,4 +1,6 @@
-{{ config(alias='solvers',
+{{ config(
+        alias='solvers',
```
Comment (gentrexha) [inline models/cow_protocol/gnosis/cow_protocol_gnosis_solvers.sql:2]: Does this need an `alias()` wrapper around it?

**Review by gentrexha — APPROVED** [review]
Left one minor comment. LGTM.

File: models/cow_protocol/gnosis/cow_protocol_gnosis_solvers.sql (hunk 1 above)
Comment (bh2smith) [reply models/cow_protocol/gnosis/cow_protocol_gnosis_solvers.sql:2]:
```suggestion
        alias=alias('solvers'),
```

**Review by jeff-dude — APPROVED** [review]
thank you 🙌

**Pull request merged**